dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:stack_trace/stack_trace.dart';
+
+import 'protocol/sentry_stack_frame.dart';
+
+class SentryStackTraceFactory {
+  const SentryStackTraceFactory();
```

Comment:
missing `inAppInclude` and `inAppExclude`

https://github.com/getsentry/sentry-java/blob/113af8211c07ef622a4e265cc6b3438a6194d9fe/sentry/src/main/java/io/sentry/MainEventProcessor.java#L28-L29

https://github.com/getsentry/sentry-java/blob/113af8211c07ef622a4e265cc6b3438a6194d9fe/sentry/src/main/java/io/sentry/SentryStackTraceFactory.java#L67-L94

I'd get the Options as param instead of 2 list as Java

Reply:
✅